### PR TITLE
Doubler83 patch 1

### PR DESCRIPTION
**Pull request by doubler83**

### CODE REVIEW

File: docs/source/roland-rath.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Hello World!
+============
+But the world is crazy.
```
Comment (gruberan21) [inline docs/source/roland-rath.rst:3]:
```suggestion
But the world is crazy.
Which is fine because we all need a little **crazy**.
```

**Review by gruberan21 — CHANGES_REQUESTED** [review]
Well done brother!

File: docs/source/roland-rath.rst [hunk 2]
```diff
@@ -1 +1,3 @@
-**Hello World!**
+Hello World!
```
Comment (BruceWayne9999) [inline docs/source/roland-rath.rst:1]: 🍡 🥇 👍 💯 🔢 🧷 🦺 🍶 ♐ 👯‍♀️ 😵‍💫

**Comment by doubler83** [comment]
okelidokeli

**Review by 09feiler — APPROVED** [review]

**Review by georguebleis — APPROVED** [review]

**Pull request merged**